docs/components/console/manage-clusters/cluster-capacity.md

```diff
@@ -0,0 +1,104 @@
+---
+id: cluster-capacity
+title: Cluster capacity
+description: "Cluster capacity provides you with a high-level overview of how well a cluster is coping with and handling its current workload."
+---
+
+import CapacityImg from './img/cluster-capacity.png';
+import CapacityPercentImg from './img/cluster-capacity-percent.png';
+
+View and manage your cluster capacity and health.
+
+## About cluster capacity
+
+Cluster capacity provides a high-level overview of how well a cluster is coping with its current workload.
+
+- Use this information to check and monitor if a cluster is appropriately sized for its workload.
+- Cluster capacity can also be used as an indicator of cluster health. For example, a cluster running at maximum capacity can be an indicator of poor cluster responsiveness.
+
+A general guideline to follow when using cluster capacity as a metric is:
+
+- **High capacity percentage**: The higher the capacity percentage value, the more likely it is that things will slow down, time out, requests will fail, and so on.
+- **Low capacity percentage**: The lower the capacity percentage value, the more you could increase the cluster workload as the cluster is probably underused.
+
+For example, if a cluster capacity is continually running at 95%, this means it is probably overloaded and may not be performing well, but if the cluster capacity is only 5% then it can probably accept more workload and may be underused.
+
+:::info
+To understand how cluster capacity is calculated, see [how cluster capacity is calculated](#capacity-calculation).
+:::
+
+## View cluster capacity
```

Comment:
:question: Perhaps we should mention here that not all clusters will have that information? "If your cluster supports capacity monitoring, the percentage is shown in..."